docs/marketplaces/stripe_connect.md

```diff
@@ -33,8 +33,11 @@ end
 @user.merchant_processor.account_link
 @user.merchant_processor.login_link
 @user.merchant_processor.transfer(amount: 25_00)
+@user.merchant_processor.onboarding_complete? # Updates via webhook based on the Stripe::Account's #charges_enabled attribute
 ```
 
+**IMPORTANT:** [Stripe has updated their account creation mechanism](https://docs.stripe.com/connect/migrate-to-controller-properties) to offer more granularity by providing a `controller` parameter. If you want to use specific `controller` options when creating an account, you'll instead need to map your desired settings to a `type` [following this Stripe guide](https://docs.stripe.com/connect/migrate-to-controller-properties#mapping-account-types-to-controller-parameters). At the moment, Pay defaults to specifying a `type` in the `Merchant#create_account` method, and both parameters cannot be sent simultaneously to Stripe. If you need to use `controller` to have more granular settings, we welcome a PR.
```

Comment:
I think it makes sense to just remove the default `type` since we can't really make any assumptions about how you plan to use it.

Reply:
Good with me, just wasn’t sure if that was important to remain or would be a breaking change for existing users

Reply:
It'll be a breaking change, but I'm alright with that.